apps/ui/src/views/Proposal.vue

```diff
@@ -154,7 +163,16 @@ watchEffect(() => {
             @get-voting-power="getVotingPower"
           >
             <h4 class="block eyebrow">Your voting power</h4>
-            <div class="pt-2">
+            <div
+              v-if="
+                votingPowerDetailsError?.details === 'NOT_READY_YET' &&
+                ['evmSlotValue', 'ozVotesStorageProof'].includes(votingPowerDetailsError.source)
+              "
+              class="mt-2"
+            >
+              Please allow few minutes for the voting power to be collected from Ethereum.
```

Comment:
Can you add the icon "information-circle" here too?

Reply:
Ok sorry just seen your comment. Best would be to show Icon inline with the text and not taking 3 line.

Reply:
You mean that icon should align to first line (because there will always be 3 lines because there is not enough space). I tried it before but it didn't look great too.

I can update the designs again to see how it looks like.

Reply:
Something like this
<img width="317" alt="image" src="https://github.com/snapshot-labs/sx-monorepo/assets/16245250/4f67264e-ea9d-4972-8ba6-3e3483e9e128">

Reply:
There is the same issue on empty messages across the app
<img width="215" alt="image" src="https://github.com/snapshot-labs/sx-monorepo/assets/16245250/18eb4b1a-4d5a-4ca3-97d3-aba6a09affbe">